### PR TITLE
TECH-13401: skaffold build workflow for flux

### PR DESCRIPTION
**Pull request by Caislear**

### CODE REVIEW

**Comment by Caislear** [comment]
https://github.com/goes-funky/workflows/pull/99 should be applied first, before merging this in.

**Comment by Caislear** [comment]
@Steffen911 thoughts around this?
I'd be for merging it as and testing it further with actual services.

File: .github/workflows/flux-build.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            } catch (error) {
+                core.setFailed(error.message);
+            }
+      - name: Download custom skaffold docker build script
+        run: curl -LsO https://raw.githubusercontent.com/goes-funky/workflows/master/scripts/docker-buildx && chmod +x docker-buildx
+      - name: Configure skaffold to build with buildkit
+        run: 'cp ./code/${{ inputs.skaffold-file }} . && yq -i ''del(.build.local) | del(.build.artifacts.[].docker) | del(.build.artifacts.[].sync.*) | .build.artifacts.[] *= {"custom": {"buildCommand": "../docker-buildx", "dependencies": {"dockerfile": {"path": "${{ inputs.docker-file }}"}}}}'' ${{ inputs.skaffold-file }}'
+      - name: Untar build artifact
+        if: inputs.untar-artifact-name
+        run: tar -xf ./code/dist/${{ inputs.untar-artifact-name }} -C ./code/dist
```
Comment (Steffen911) [inline .github/workflows/flux-build.yaml:85]: You're not download this anywhere, do you?

**Review by Steffen911 — APPROVED** [review]
Did you test it somewhere? In that case it's fine. It's hard to review GHA 😄

File: .github/workflows/flux-build.yaml (hunk 1 above)
Comment (Caislear) [reply .github/workflows/flux-build.yaml:85]: Probably can drop the whole untar build artifact, doesn't really add any value to this workflow anyway. Will run through and just nuke everything that is not needed to keep it bare functionality.

**Comment by Caislear** [comment]
> Did you test it somewhere? In that case it's fine. It's hard to review GHA 😄

yeppers on the job-monitor before nuking it away :)

**Pull request merged**